magpie/alembic/versions/3cfc41c4a5f0_groups_pkey_change.py

```diff
@@ -6,8 +6,7 @@
 Create Date: 2012-06-27 02:15:58.776223
 """
 from __future__ import unicode_literals
-# noinspection PyUnresolvedReferences
-from alembic.context import get_context
+from alembic.context import get_context  # noqa: F401
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'alembic.context'](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)